src/components/users/TickButton.tsx

```diff
@@ -1,3 +1,4 @@
+'use client'
```

Comment:
I was getting errors locally telling me to add this here. "You're importing a component that needs useState. It only works in a Client Component but none of its parents are marked with "use client", so they're Server Components by default. Learn more: https://nextjs.org/docs/getting-started/react-essentials"

I don't know if thats a local-only issue for me though.

Reply:
We do want to address that error.  Generally, we want to have as many server-side components as possible and add "use client" directive as escape hatch.  Maybe don't introduce "hover" behavior as it only works on desktop.

Reply:
I'm happy to remove the hover functionality, but I don't think thats related to the 'use client' error.

Prior to my code changes, the TickButton relied on `useState` to determine if a route has been ticked by a user, so it knows whether to display `Tick This Route` or `View Ticks` on the button.

I'm not sure how to rewrite that functionality without useState, but i'm not at all an expert on Next.js.

I do see several other components that 'use client', so i'm not sure what the best solution is here.